style_guide.md

```diff
@@ -1,26 +1,100 @@
-# BODS Style Guide - #draft
+# BODS Style Guide - #WIP
+
+This style guide is a draft to be discussed and not yet implemented across the BODS schema and documentation (as of May 2019, BODS v0.2). It can be used to guide editing and content writing.
+## Common Conventions
+
+### Readability 
+We try to write clear, concise documentation, using plain English where possible. 
+
+These tools can help with this:
+- [Hemingway editor](https://hemingwayapp.com/)
+- [Grammarly](https://www.grammarly.com/)
+- [Readability tools](https://www.webfx.com/tools/read-able/)
+
+### Spelling 
+Use English spelling:
```

Comment:
@StephenAbbott do you know whether OO has a preference for UK or US spelling? On the OO website I can see examples of both. Currently the BODS documentation is majority UK so I kept with that as a starting point as it would require less changes to the docs to make it consistent.

Reply:
UK spelling is preferred in the OO style guide